File: src/main/java/design_patterns/structural/flyweight/TestFunctions.java
package design_patterns.structural.flyweight;

public class TestFunctions {

    public static final String colors[] = {"Red", "Green", "Blue", "White", "Black"};

    public static String getRandomColor() {
        return colors[(int) (Math.random() * colors.length)];
    }

    public static int getRandomX() {
        return (int) (Math.random() * 100);
    }

    public static int getRandomY() {
        return (int) (Math.random() * 100);
    }
}

File: src/main/kotlin/design_patterns/structural/flyweight/TestFunctions.kt
package design_patterns.structural.flyweight

object TestFunctions {
    val colors = arrayOf("Red", "Green", "Blue", "White", "Black")
    val randomColor: String
        get() = colors[(Math.random() * colors.size).toInt()]
    val randomX: Int
        get() = (Math.random() * 100).toInt()
    val randomY: Int
        get() = (Math.random() * 100).toInt()
}